sections/sprint-4/test-07-05.tex

```diff
@@ -47,6 +47,11 @@ \subsubsection{The results of the test}
 When this was fixed, a new bug was encountered when trying to send the host configuration data to the players, where some type conversions were missing that transformed the user input from strings to the appropriate format for calling the Pozyx library.
 From the host program, there was also a minor usability issue where there was no specific information regarding the unit which should be used when inputting information about anchor positions, so the user would not know whether to use centimeters, millimeters or meters, which could cause some confusion.
 Finally, once four players were able to connect and receive configuration data from the host, the game launched for the players, but they were immobile, not moving on the field based on the Pozyx data.
+\\\\
 Another issue encountered during testing was related to the blue goal zone.
 This goal zone specifically could flicker when the player moved his head, which the red zone did not.
-As such, while the test did not achieve the original goal, it still exposed certain issues to fix for the upcoming sprint.
\ No newline at end of file
+As such, while the test did not achieve the original goal, it still exposed certain issues to fix for the upcoming sprint.
+Further research into this bug revealed that it was another problem with the Pozyx system, where it would repeatedly report $(0,0,0)$ positions with actual positioning data in between.\\
+Most of these Pozyx issues had not previously been identified since we, up until this point, had been using mocked data, and had not set up the Pozyx since conducting the experiments in \autoref{sprint1:experiment}.
```

Comment:
```suggestion
Most of these Pozyx issues had not previously been identified since we, up until this point, had been using mock data, and had not set up the Pozyx since conducting the experiments in \autoref{sprint1:experiment}.
```